README.md

```diff
@@ -117,6 +117,134 @@ class Test {
 ```
 
 
-## Future work
-  - Converters for `java.util.stream`
-  - [`Spliterator`](https://docs.oracle.com/javase/8/docs/api/java/util/Spliterator.html)s for Scala collections
+## Converters from Scala collections to Java 8 Streams
+
+Scala collections gain `seqStream` and `parStream` as extension methods that produce a Java 8 Stream
+running sequentially or in parallel, respectively.  These are automatically specialized to a primitive
+type if possible.  For instance, `List(1,2).seqStream` produces an `IntStream`.  Maps additionally have
+`seqKeyStream`, `seqValueStream`, `parKeyStream`, and `parValueStream` methods.
+
+Scala collections also gain `accumulate` and `stepper` methods that produce utility collections that
```

Comment:
When would you use these methods? It looks like `accumulate` always copies the collection sequentially, which does not seem useful to me as a separate feature. For sequential streams, I'd expect that I could just use the original collection, and for parallel streams I'd want the implementation to decide whether copying is really necessary or the original Scala parallel collection (or even a sequential collection like ArrayBuffer) can be used efficiently. Building an accumulator sequentially could still be done with `to[Accumulator]`, which would probably set the right expectations for performance characteristics.